requirements/base.txt

```diff
@@ -6,7 +6,7 @@
 #
 asgiref==3.7.2
     # via django
-backports-zoneinfo==0.2.1
+backports-zoneinfo==0.2.1; python_version < '3.9'
```

Comment:
Did the python version restriction happen because of of an update to a constraint or input file? or because we ran `make upgrade` with a newer python version? or something else?

Reply:
It was supposed to be part of `constraints.txt` which I missed. I've added it here now https://github.com/openedx/edx-notes-api/pull/392/commits/a388a846e38bc6308dffb7724028ac47b4001404